src/main/java/uk/ac/bath/se/SplashScreen.java

```diff
@@ -7,6 +7,19 @@
 
 class SplashScreen {
 
+    //this stArea is for saving the user name created by the player and past it to the DungeonView.
+    // so the game can save the historical records
+
+    static String stArea;
```

Comment:
I don't think this should be static as it has no default value

Reply:
I tried to make it no static in the first time, but the dungeon cannot accept it.